doc/lua_api.md

```diff
@@ -5466,21 +5466,44 @@ Call these functions only at load time!
     * Returns an integer object handle uniquely identifying the registered
       ore on success.
     * The order of ore registrations determines the order of ore generation.
+* `minetest.override_ore(name, ore definition)`
+    * Override existing ore.
+* `minetest.unregister_ore(name)`
+    * Unregisters the decoration from the engine, and deletes the entry with key
+      `name` from `minetest.registered_decorations`.
```

Comment:
Function not necessary if we'd accept an explicit `nil` in `minetest.override_ore` to remove it. Or is that bad practice?

Reply:
There are already functions like `minetest.unregister_item(name)` in Lua API. 
So keeping of same Lua API style is a good idea, I believe.